src/dataServices/dynamoDbHelper.ts

```diff
@@ -38,19 +57,12 @@ export default class DynamoDbHelper {
     }
 
     async getMostRecentValidResource(resourceType: string, id: string): Promise<GenericResponse> {

```

Comment:
A bit retrospective here is there a better term than `valid` here? is not that 'DELETED' resources are invalid. I can't think of another name but if you have a better one let me know

Reply:
Thoughts on this name: `getMostRecentUserReadableResource`?

I'm also adding comments to the method to make it clearer.
```
/**
  * @return The most recent resource that has not been deleted and has been committed to the database (i.e. The resource is not in a transitional state)
  */
```

Reply:
I like good change